server/service/endpoint_teams.go

```diff
@@ -220,3 +220,20 @@ func makeTeamEnrollSecretsEndpoint(svc fleet.Service) endpoint.Endpoint {
 		return teamEnrollSecretsResponse{Secrets: secrets}, err
 	}
 }
+
+type modifyTeamEnrollSecretsRequest struct {
+	TeamID  uint                 // From request path
+	Secrets []fleet.EnrollSecret `json:"secrets"`
+}
+
+func makeModifyTeamEnrollSecretsEndpoint(svc fleet.Service) endpoint.Endpoint {
```

Comment:
Ah, when I saw you were using the previous format for endpoints, it was fine because I thought it was going to be inside the `ee` package, and we haven't really touched that. But now I see it's in the regular service, so we should use the new format.

Happy to hop on a call and discuss what should change. Apologies for not realizing this sooner.